src/types.ts

```diff
@@ -110,6 +124,7 @@ export type FormatToken =
   | "A"
   | "ZZ"
   | "Z"
+  | "SSS"
```

Comment:
Hmm, 🤔 I dont see any code in this commit that adds `SSS` as a supported token to the parse/format functions so I think we should not add it to the base FormatToken just yet.